install.md

```diff
@@ -11,27 +11,46 @@ weight: 1
 Install [Node](https://nodejs.org/en/download/) for your platform, then clone and build the Eve starter repository:
```

Comment:
Let's include a link here to the eve starter repo, theoretically someone without git could download just a zip file here, or people who prefer ssh can grab the ssh url.